messaging/kafka-producer/src/test/java/io/quarkus/ts/messaging/kafka/producer/OpenShiftKafkaSnappyIT.java

```diff
@@ -21,7 +20,6 @@
 import io.vertx.mutiny.core.buffer.Buffer;
 
 @OpenShiftScenario
-@DisabledOnNative(reason = "In native mode, Snappy is disabled by default as the use of Snappy requires embedding a native library and unpacking it when the application starts.")
```

Comment:
Have been a changed in upstream? docs?

Reply:
it's documented here: https://quarkus.io/version/main/guides/kafka#using-snappy-for-message-compression and added here https://github.com/quarkusio/quarkus/pull/15174 in Quarkus 1.13